docker-compose.yml

```diff
@@ -10,6 +9,8 @@ services:
         volumes:
             - .:/code
         working_dir: /code
+        environment:
+          PHP_VERSION: ${PHP_VERSION}
```

Comment:
This can be a fixed value I guess, or at least provide a default value. Could you change this?

Great job on these changes btw!

Reply:
Yes, i can set default value, but env variable `PHP_VERSION` set in travis script build

Reply:
Also in Docker file i popose install of composer to replace on this construction
`
FROM composer as composer-build #official docker image with composer (https://github.com/composer/docker/blob/master/1.6/Dockerfile)
COPY --from=composer-build /usr/bin/composer /usr/local/bin/composer
`